pages/preferences-form.js

```diff
@@ -0,0 +1,27 @@
+import PreferencesButton from "@/components/PreferencesButton";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function PreferencesForm() {
+  const availableLocations = [
+    "I'm stuck in a boat",
+    "I'm in a forest",
+    "I'm in a city",
+    "I'm in the mountains",
+  ];
+  const [selectedLocations, setSelectedLocations] = useState([]);
+
+  return (
+    <>
+      <h3>I am bored, and...</h3>
+      {availableLocations.map((location, i) => (
+        <PreferencesButton
+          key={location}
+          value={location}
+          selectedPreferences={selectedLocations}
+          setPreferences={setSelectedLocations}
+        />
+      ))}
+    </>
+  );
```

Comment:
Here as well a bit confusing and I am not sure where you are trying to get to. You don't seem to be working wiht these values like selectedPrefereces or selected Locations. Btw, why do you use two props and what is the difference?

```suggestion
  return (
    <>
      <h3>I am bored, and...</h3>
      {locations.map((location) => (
        <PreferencesButton
          key={location}
          value={location}
          isPreference={isPreference}
        />
      ))}
    </>
  );
```

Reply:
Im nächsten Schritt kannst du dann mit der Information "isPreference === true"... weiterarbeiten. Am besten in einem Form element :)